### PR TITLE
Track unrecognized URLs

### PR DESCRIPTION
**Pull request by tchayen**
Adding logging for URLs that app failed to recognize. It will help us debugging future problems with deeplinking.

### CODE REVIEW

File: src/handlers/deeplinks.ts [hunk 1]
```diff
@@ -97,6 +99,9 @@ export default async function handleDeeplink(
               address: addressOrENS,
             });
           } else {
+            logger.sentry('Failed to handle url', url);
+            const error = new Error('Invalid deeplink');
+            captureException(error);
```
Comment (osdnk) [inline src/handlers/deeplinks.ts:104]:
```suggestion
            const error = new Error('Invalid deeplink: ' + url);
            captureException(error);
```


hb this?

Comment (tchayen) [reply src/handlers/deeplinks.ts:104]: Looks better to me. But I am not an expert in our sentry and logging config. @brunobar79 pls double check.

**Review by brunobar79 — APPROVED** [review]
👍

File: src/handlers/deeplinks.ts [hunk 2]
```diff
@@ -21,6 +22,7 @@ import {
 import { ETH_ADDRESS } from '@rainbow-me/references';
 import Routes from '@rainbow-me/routes';
 import { ethereumUtils } from '@rainbow-me/utils';
+import logger from 'logger';
```
Comment (brunobar79) [inline src/handlers/deeplinks.ts:25]: You're importing logger but you're not using it anymore

**Review by osdnk — APPROVED** [review]
ok

**Pull request merged**